### DISCUSSION TITLE
Cache Settings and Observe changes

### DISCUSSION THREAD

**Discussion by Zakarialabib** [Zakarialabib/myStockMaster#12] [Ideas]
@elmarzouguidev suggested Cache Settings and Observe changes

**Comment by ismaail** [comment]
the application in general is super slow (3s on my local machine)

I tried caching some redundant DB queries (check with laravel-debugbar), it didn't help!

**Reply by Zakarialabib** [reply]
I checked the online version yesterday, which loads between 300 and 600ms,
My mac setup loads between 15 and 20sec🤣

**Reply by ismaail** [reply]
what's the online version link ?

**Reply by Zakarialabib** [reply]
https://test.hotechserver.com

**Reply by ismaail** [reply]
thanks, I can't login, can u send me the credentials (via Facebook messenger),

and don't forget to disable the **debugbar** & `APP_DEBUG=true` in `.env`.

**Reply by ismaail** [reply]
I restared php-fpm Docker container without xdebug and now it runs very fast **64ms** 😅

with xdebug enabled: **1.36s** to **3s**.

![image](https://user-images.githubusercontent.com/4411670/207467483-67a1cc83-6690-41bd-9d70-1d7cc28a105a.png)

**Reply by Zakarialabib** [reply]
Nice good to hear that
